client/src/actions/code.js

```diff
@@ -74,6 +83,23 @@ export function fetchCode(id) {
     }
 }
 
+export function fetchCodesByUserId(userid) {
+    return function (dispatch) {
+        return request
+            .get('http://localhost:8000/app/codes')
```

Comment:
Does this not fail? It doesn't have the slash at the end, which we've run into problem before.